x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -544,6 +544,19 @@ export const getCatchupTuples = ({
   return catchupTuples;
 };
 
+/**
+ * Takes the rule schedule fields `interval` and `lookback` and uses them to calculate the `from` value for a rule
+ *
+ * @param interval moment.Duration representing the interval on which the rule runs
+ * @param lookback moment.Moment representing the rule's additional lookback
+ * @returns moment.Moment representing the rule's 'from' property
```

Comment:
function signature points params are strings and return value is number, not moment objects